### PR TITLE
docs(webapps): adds authentication cache time to live docs

### PR DESCRIPTION
**Pull request by tasso94**
related to camunda/camunda-bpm-platform#3165

### CODE REVIEW

File: content/user-guide/spring-boot-integration/configuration.md [hunk 1]
```diff
@@ -776,6 +776,30 @@ When setting to <code>/</code>, the legacy behavior of Camunda Spring Boot Start
   <td><code>max-age=31536000</code></td>
 </tr>
 
+<tr id="auth-cache">
+  <td rowspan="3"><code>camunda.bpm.webapp.auth.cache</code></td>
+</tr>
+<tr>
+  <td><code>.ttl-enabled</code></td>
```
Comment (tasso94) [inline content/user-guide/spring-boot-integration/configuration.md:783]: I'll change this property in the implementation as well.

File: content/user-guide/spring-boot-integration/configuration.md [hunk 2]
```diff
@@ -776,6 +776,30 @@ When setting to <code>/</code>, the legacy behavior of Camunda Spring Boot Start
...
+    Allowed set of values is <code>true</code> and <code>false</code>.
+  </td>
+  <td><code>true</code></td>
+</tr>
+<tr>
+  <td><code>.time-to-live</code></td>
+  <td>
+    The allowed set of values:
+    <ul>
+      <li>a time duration in milliseconds between <code>1</code> and <code>2<sup>63</sup>-1</code></li>
```
Comment (yanavasileva) [inline content/user-guide/spring-boot-integration/configuration.md:795]: I suggest to reword the description. Emphasizing what the property does and what type it is. So if a user is reading it as a standalone property (without paying attention to the `ttl-enabled`), they have enough information how to adjusted it according to their needs.
```suggestion
    An amount of milliseconds, the cache is used for an HTTP session by the Webapps before they recreate it and query for the authentication information again from the database. The allowed set of values:
    <ul>
      <li>a time duration in milliseconds between <code>1</code> and <code>2<sup>63</sup>-1</code></li>
```

**Review by yanavasileva — APPROVED** [review]
New docs and adjustments look good. I added two suggestions for consideration.

File: content/update/patch-level.md [hunk 3]
```diff
@@ -384,6 +384,18 @@ Tenant information is populated for User operation logs from 7.18.5 onwards, use
 
 In case you want to avoid tenant checks, please refer to [Disable the transparent access restrictions]({{< ref "/user-guide/process-engine/multi-tenancy.md#disable-the-transparent-access-restrictions" >}}).
 
+## 7.18.7 to 7.18.8 / 7.17.10 to 7.17.11 / 7.16.17 to 7.16.18
```
Comment (yanavasileva) [inline content/update/patch-level.md:387]: I assume the 7.18 patch will be 7.18.6:
```suggestion
## 7.18.5 to 7.18.6 / 7.17.10 to 7.17.11 / 7.16.17 to 7.16.18
```

**Review by yanavasileva — APPROVED** [review]
Thank you for applying the feedback. Please double check the patch versions before merge and backport.

File: content/webapps/shared-options/authentication.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+## Cache
+
+By default, after a successful login, the web apps keep a copy of the authentication information in memory for five minutes. This information is cached, and therefore we call this behavior **authentication cache**. 
+
+The authentication cache is a performance optimization to prevent performing for each REST API request 
+multiple database queries that potentially retrieve the same authentication information repeatedly
+given it didn't change.
+
+Read the security implications of the [authentication cache]({{< ref "/user-guide/security.md#authentication-cache" >}}) in our Security Instructions.
```
Comment (christinaausley) [inline content/webapps/shared-options/authentication.md:40]: Should we link to the **Security Instructions** here as well?

**Review by christinaausley — APPROVED** [review]
Technical review in latest commit, but largely just adjusted webapps to web apps. One non-blocking comment as an opportunity to link to the section we mention.

File: content/webapps/shared-options/authentication.md [hunk 5]
```diff
@@ -24,32 +24,31 @@ The authentication information consists of the following:
...
 they recreate it and query for the authentication information again from the database.
 
 You can change the `cacheTimeToLive` configuration property as follows:
 
-* Set the parameter value to a time duration in milliseconds between `1` and <code>2<sup>63</sup>-1</code>.
-* Set the parameter value to `0`, effectively leading to querying for the authentication information on each REST API request.
-* Remove the `<init-param>...</init-param>` entirely or change the parameter value to `<param-value/>` to keep the cache for the lifetime of the HTTP session.
+1. Set the parameter value to a time duration in milliseconds between `1` and <code>2<sup>63</sup>-1</code>.
+2. Set the parameter value to `0`, effectively leading to querying for the authentication information on each REST API request.
+3. Remove the `<init-param>...</init-param>` entirely or change the parameter value to `<param-value/>` to keep the cache for the lifetime of the HTTP session.
```
Comment (tasso94) [inline content/webapps/shared-options/authentication.md:51]: Almost all of your changes make sense to me. Thank you 🙂 👍 
However, this is actually a list with no clear order. I'll revert this back to unordered list and change the introducing sentence a bit.
⬇️ 

```
You can change the `cacheTimeToLive` configuration property with the following allowed values:

* Set the parameter value to a time duration in milliseconds between `1` and <code>2<sup>63</sup>-1</code>.
* Set the parameter value to `0`, effectively leading to querying for the authentication information on each REST API request. 
* Remove the `<init-param>...</init-param>` entirely or change the parameter value to `<param-value/>` to keep the cache for the lifetime of the HTTP session.
```

**Review by christinaausley — APPROVED** [review]

**Comment by christinaausley** [comment]
I am too used to pressing GO in `camunda-platform-docs` and merged this when I should have requested @tasso94 😀. Adjusted in https://github.com/camunda/camunda-docs-manual/pull/1408.

**Comment by tasso94** [comment]
No worries, @christinaausley! 🙂 Thank you for your follow-up PR.

**Pull request merged**